### DISCUSSION TITLE
Getting invalid request when logging in

### DISCUSSION THREAD

**Discussion by Pahuts** [singpass/demo-app#28] [Q&A]
Hi everyone,

I'm getting an invalid request error when logging in the demo app. I adjusted the demo app to use my client id and trying to login with the singpass provided user credentials. Is there something I configured incorrectly?

**Comment by Pahuts** [comment]
We were able to get the access token for the user but we still can't fetch the user information.

{"access_token":"eyJhbGciOiJFUzI1NiIsInR5cCI6ImF0K2p3dCIsImtpZCI6ImFsaWFzL3N0Zy1zcC1hdXRoLWFwaS1pZC10b2tlbi1zaWduaW5nLWtleS1rbXMtYXN5bW1ldHJpYy1rZXktYWxpYXMifQ.eyJzdWIiOiJ1PTllNTRmOTU2LTk3MjItNDdiZS1iNGFmLTAzNzhjYWYwNzI1OSIsImNsaWVudF9pZCI6IkNoR0RJWEZ5NEgwVjR5YWFobVd2eks0NU5TNHBaNGtmIiwic2NvcGUiOiJvcGVuaWQgdWluZmluIG5hbWUiLCJqdGkiOiJhdC1JMjJtVllnN3NQSkktOThuY1ltNXE0QWFXRXczeDVXSUN1WDd5VG9XblZrIiwiaWF0IjoxNzM0NDI0NTk5LCJleHAiOjE3MzQ0MjYzOTksImF1ZCI6Imh0dHBzOi8vc3RnLWlkLnNpbmdwYXNzLmdvdi5zZy91c2VyaW5mbyIsImlzcyI6Imh0dHBzOi8vc3RnLWlkLnNpbmdwYXNzLmdvdi5zZyJ9.xY3wXbuNfkIyYH79T2iqsJwzbXJyMWDqwIFwP0YwL88chwzCpZXa6AsufjBoVzSoDZTkKnQ2y_TukV0ZSKq5kA","token_type":"Bearer","id_token":"eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImFsaWFzL3N0Zy1zcC1hdXRoLWFwaS1pZC10b2tlbi1zaWduaW5nLWtleS1rbXMtYXN5bW1ldHJpYy1rZXktYWxpYXMifQ.eyJzdWIiOiJ1PTllNTRmOTU2LTk3MjItNDdiZS1iNGFmLTAzNzhjYWYwNzI1OSIsImF1ZCI6IkNoR0RJWEZ5NEgwVjR5YWFobVd2eks0NU5TNHBaNGtmIiwiYW1yIjpbInB3ZCJdLCJub25jZSI6ImIxYWI5Yzk4LTlhMTktNDEyOS05OThiLTViMjM2OWFiMDc2NiIsImlhdCI6MTczNDQyNDU5OSwiZXhwIjoxNzM0NDI1MTk5LCJpc3MiOiJodHRwczovL3N0Zy1pZC5zaW5ncGFzcy5nb3Yuc2cifQ.3LYYI_aWj8-QpSp-ZyfrY3rhLjpqf8kw4A7Xk-UM-H0JT6L1HsDDNP8uUgNmOUK6LPoAuB6PMivvRTDCIFepaQ"}


**ERROR MESSAGE**
  error: 'invalid_request',
  error_description: 'An error has occurred.'

**Reply by trungdoublelift** [reply]
Hi, i am currently facing the same issue have you found a fix for it yet ?

**Comment by yuensunn** [comment]
I had a similar issue and raised an enquiry at https://partnersupport.singpass.gov.sg/.

Apparently, it's failing to retrieve user info because you are probably trying to access a staging account without a test profile.

You should try using one of the accounts provided [here](https://docs.developer.singpass.gov.sg/docs/testing/myinfo-test-personas).

**Comment by kienwoon** [comment]
username: G4440433N
password: Userinfo@2024

reference:
https://docs.developer.singpass.gov.sg/docs/testing/myinfo-test-personas
